File: Week-1/Day-3/Ishika/4.java
import java.util.*;
class leap
{
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);

        int t,y,i;
        t=sc.nextInt();
        for(i=1;i<=t;i++)
        {
            y=sc.nextInt();
            if(leap(y)==1)
                 System.out.println("It is a leap year.");
            else 
            {
                for(int j=1;j<=10;j++)
                {
                    y++;
                    if(leap(y)==1)
                    {
                        
                        System.out.println("Not a leap year. Suggested: "+y);
                        break;
                    }
                }
            }
        }
        
    }

    static int leap(int y)
    {
        if(y%400==0)
            return 1;
        else if(y%100 ==0)
            return 0;
        else if(y%4==0)
            return 1;
        else
            return 0;
    }
   
}

/* run this program using the console pauser or add your own getch, system("pause") or input loop */

File: Week-1/Day-4/Ishika/4.cpp
#include<iostream>
using namespace std;

/* run this program using the console pauser or add your own getch, system("pause") or input loop */

int main()
{
	int n;
	cin >> n;long long a[n],c;
	for(int i=0;i<n;i++)
	cin >> a[i];
	for(int i=0;i<3;i++)
	{
	c=0;
	for(int j=i;j<n;j+=3)
	c=c+a[j];
	cout  << c << " ";
	}
	return 0;
}

